### DISCUSSION TITLE
[Feature Request]: ALDOC -> add property to include local or internal methods/events/codeunit

### DISCUSSION THREAD

**Discussion by ArvinNube** [microsoft/AL-Go#1600] [Ideas]
### Feature description

hello, I saw that DocFx has a property to make event methods and other visible that are private (includePrivateMembers). Is it possible to include this parameter in ALDOC as well?
Thanks

**Comment by JPO-prodware** [comment]
Hello ArvinNube. Is your feature request solved?

**Comment by ArvinNube** [comment]
@JPO-prodware  No they haven't responded yet

**Comment by freddydk** [comment]
Sorry, missed this one - @esbenk, do you know whether ALDoc has this capability?

**Comment by freddydk** [comment]
Will look at this together with a few other ALDOC enhancements.

**Comment by elweilando** [comment]
Is there anything new here? 
DocFX has a corresponding setting to include private members in the documentation: [https://dotnet.github.io/docfx/reference/docfx-json-reference.html#includeprivatemembers](https://dotnet.github.io/docfx/reference/docfx-json-reference.html#includeprivatemembers)
